sky/shell/tracing_controller.cc

```diff
@@ -138,7 +153,9 @@ void TracingController::StopTracingAsync(const base::FilePath& path,
 }
 
 void TracingController::StartDartTracing() {
-  Dart_GlobalTimelineSetRecordedStreams(~0);
+  if (dart_initialized_) {
```

Comment:
I'd probably write this as early-return instead.  Allows the method to grow more later with ease.

if (!dart_initialized_)
  return;